src/main/scala/com/monsanto/arch/cloudformation/model/resource/Lambda.scala

```diff
@@ -0,0 +1,92 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model._
+import spray.json.{JsString, JsValue, JsonFormat, DefaultJsonProtocol}
+
+class Runtime(val runtime: String)
```

Comment:
I have not used Lambda for almost 1 year. 

However, if we know the universe of `runtime` values, then I believe using an [algrebraic data type](http://stackoverflow.com/questions/26689951/writing-algebraic-data-type-in-scala) should be considered.

```
sealed trait Runtime
case object Java8 extends Runtime
case object ...
```